.github/.gitmessage.txt

```diff
@@ -0,0 +1 @@
+# <type>(scope): <short description>
```

Comment:
@misterabdullahAziz as we discussed, can have some examples in here (similar to the ones in the doc you showed us during the retrospective).